### ISSUE TITLE
Feature Request??

### ISSUE THREAD

**Issue by RhoiDath** [brittyazel/Neuron#318]
Hi, Long time user of this chain of addons all the way back to Vanilla.

Ive just started classic and the bars are working brilliantly so far, however I was just wondering if there was a way to make the tooltip not appear at the cursor when hovering over the bars. I find the tooltip is getting in the way of other things that I would like to be able to see at the same time.

Ive discovered it is because the buttons are macros (at least I think thats why). If I have the blizz bars up, the tooltip appears where I want, but if it is a macro it appears on the cursor.

does that make sense?

TLDR: the tooltip appears at the cursor, any way to change this to a fixed location?

Thanks

Edit - Ohh I just remembered something else. My cast bar is fine for casting spells, but when I am channeling things like Drain Soul for example, nothing appears.

**Comment by brittyazel** [comment]
Yep in planning on adding something for the tooltips. There's already an issue open for that so I'm going to go ahead and close this one. Please comment on the other ticket with any suggestions how you would like it implemented. Also please open a separate ticket for channeling issue

**Comment by brittyazel** [comment]
Yep in planning on adding something for the tooltips. There's already an issue open for that so I'm going to go ahead and close this one. Please comment on the other ticket with any suggestions how you would like it implemented. Also please open a separate ticket for channeling issue with as much info as you can provide
